docs/appendices/release-notes/5.10.3.rst

```diff
@@ -53,3 +53,7 @@ Fixes
 
 - Fixed an issue that prevented the use of custom analyzers with
   :ref:`shingle token filters<shingle-tokenfilter>`.
+
+- Fixed an issue that would prevent usage of a column with a custom
+  :ref:`ANALYZER<ref-create-analyzer>` which has been added to a table with
```

Comment:
```suggestion
  :ref:`ANALYZER <ref-create-analyzer>` which has been added to a table with
```

Should this also note that it doesn't repair tables where this was already done and that it only works for new `ALTER TABLE .. ADD COLUMN` statements?

Reply:
yes, thx for the reminder.